src/client/apps/edit/components/header/index.tsx

```diff
@@ -195,17 +203,67 @@ export class EditHeader extends Component {
             {this.getSaveText()}
           </SaveButton>
 
-          <a href={`${forceURL}/article/${article.slug}`} target="_blank">
-            <button className="avant-garde-button">
+          <Link href={`${forceURL}/article/${article.slug}`} target="_blank">
+            <RightHeaderButton className="avant-garde-button">
               {article.published ? "View" : "Preview"}
-            </button>
-          </a>
-        </div>
-      </div>
+            </RightHeaderButton>
+          </Link>
+        </Flex>
+      </EditHeaderContainer>
     )
   }
 }
 
+const EditHeaderContainer = styled(Flex)`
+  justify-content: space-between;
+  padding: ${space(1)}px;
+`
+
+const HeaderButton = styled.button`
+  border-radius: 0;
+  padding: 11px 18px;
+
+  &[data-disabled="true"] {
```

Comment:
Do we need the data attributes for anything besides color? If not, may be better to use boolean props for `isDisabled` and `isActive`